validator/src/main.rs

```diff
@@ -3294,6 +3273,18 @@ fn get_deprecated_arguments() -> Vec<Arg<'static, 'static>> {
             .hidden(true)
             .takes_value(false)
             .help("Disable manual compaction of the ledger database (this is ignored)."),
+        Arg::with_name("rocksdb_compaction_interval")
```

Comment:
I moved the arguments to a retired function. This keeps them from cluttering the actual code without actually breaking anyone. 

It'd be nice to rip them completely; however, I'm fairly satisfied with them here such that they're no longer cluttering the actual code. Maybe we defer on ripping them out, and we can instead do a mass retiring of arguments at some point in the future if we really want to